### ISSUE TITLE
Infinite loop in PUG preprocessing

### ISSUE THREAD

**Issue by zombiegh0st** [chinchang/web-maker#272]
Typing a for loop end up in infinite loop due to auto refresh at 
`- for (i=0; i<5; )`
I think because there is no i++ it cause an infinite loop.

**Comment by chinchang** [comment]
Yes. That's the reason. And currently pug code isn't instrumented in Web Maker to prevent such infinite loops.
